src/http/modules/ngx_http_ssl_module.c

```diff
@@ -788,9 +788,18 @@ ngx_http_ssl_merge_srv_conf(ngx_conf_t *cf, void *parent, void *child)
     if (conf->verify) {
 
         if (conf->client_certificate.len == 0 && conf->verify != 3) {
-            ngx_log_error(NGX_LOG_EMERG, cf->log, 0,
-                          "no ssl_client_certificate for ssl_verify_client");
-            return NGX_CONF_ERROR;
+            if (conf->protocols & (NGX_SSL_SSLv3|NGX_SSL_TLSv1))
```

Comment:
Avoided check for SSLv2 and added for SSLv3 TCLv1, since mTLS is not supported with SSLv2. 
Considering if  "ssl_protocols SSLv2 TLSv1.3;" is configured, then config should react as per TLS1.3 for ssl_client_certificate.